docker/dockerfile-base

```diff
@@ -0,0 +1,48 @@
+FROM alpine:latest
+
+ENV ELM_VERSION 0.19.1
+ENV SCALA_VERSION 2.12.6
+ENV SBT_VERSION 1.2.1
```

Comment:
Docspell uses currently sbt 1.3.13. But sbt will download the correct version anyways on build. So it doesn't really matter, but it also is good to use the new version here.